contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-api.mdx

```diff
@@ -0,0 +1,88 @@
+## Step 1: Evaluate the feature flag value using the `/decide`
+
+`/decide` is the endpoint used to determine if a given flag is enabled for a certain user or not.
+
+#### Request
+
+```shell
+curl -v -L --header "Content-Type: application/json" -d '  {
+    "api_key": "<ph_project_api_key>",
+    "distinct_id": "distinct_id_of_your_user",
+    "groups" : { # Optional
+        "group_type": "group_id" # Replace "group_type" with the name of your group type. Replace "group_id" with the id of your group.
+    },
+    "person_properties": {"<personProp1>": "<personVal1>"}, # Optional. Include any properties used to calculate the value of the feature flag.
+    "group_properties": {"group type": {"<groupProp1>":"<groupVal1>"}} # Optional. Include any properties used to calculate the value of the feature flag.
+}' https://app.posthog.com/decide?v=3
+```
+
+> **Note:** `person_properties` and `group_properties` are optional. 
+> 
+> By default, PostHog will use the user and group properties stored in PostHog. You only need to provide `person_properties` and `group_properties` if you wish to use different values than the ones in PostHog.
+
+#### Response
+
+```shell
+{
+  "config": {
+    "enable_collect_everything": true
+  },
+  "editorParams": {},
+  "errorComputingFlags": false,
+  "isAuthenticated": false,
+  "supportedCompression": [
+    "gzip",
+    "lz64"
+  ],
+  "featureFlags": {
+    "my-awesome-flag": true,
+    "my-awesome-flag-2": true,
+    "my-multivariate-flag": "some-string-value",
+    "flag-thats-not-on": false,
+  },
+  "featureFlagPayloads": {
+    "my-awesome-flag": "example-payload-string",
+    "my-awesome-flag-2": {"color": "blue", "animal": "hedgehog"}
+  }
+}
+```
+
+> **Note:** `errorComputingFlags` will return `true` if we didn't manage to compute some flags (For example, if there's an [ongoing incident](https://status.posthog.com/)). 
+> 
+> This allows for making partial updates to currently active flags in your clients.
+
+import IncludePropertyInEvents from "./include-feature-flag-property-in-backend-events.mdx" 
```

Comment:
also worth noting that you don't have to do this manually in backend SDK always. There's an option to `send_feature_flags_with_event` in most SDKs. The downside here though is that this makes an additional request to `/decide` to get feature flag values, which can slow things down on your backend, unlike doing this manually where ideally you're already using local evaluation to compute flags.

Reply:
Thanks @neilkakkar . Is the default value for `send_feature_flags_with_event` false?

Which way do we recommend more?

Reply:
default false yes, recommendation depends on your use case, if you don't care about the extra API call & your server is long-lived, definitely just use the param.

In all other cases (which is probably most people's backend, do it manually)